docs/book/_toc.yml

```diff
@@ -90,12 +90,21 @@
   - file: backends/using-docker
     title: Using Docker
 
+- part: Benefits
+  chapters:
+    - file: benefits/extensibility
+      title: Extensibility
+    - file: benefits/ensuring-ml-reproducibility
+      title: Reproducibility
+    - file: benefits/tracked-metadata
+      title: Metadata Tracking
+
 - part: Tutorials
   chapters:
     - file: tutorials/creating-first-pipeline.ipynb
       title: Creating your first pipeline
     - file: tutorials/running-a-pipeline-on-a-google-cloud-vm
-      title: Running a training pipeline on a Google Cloud VM
+      title: Running a pipeline on Google Cloud VM
     - file: tutorials/running-a-pipeline-on-kubernetes
       title: Running a training pipeline on Kubernetes
```

Comment:
It might look a bit better if we kept the wording within the titles consistent. ("pipeline" or "training pipeline")